### ISSUE TITLE
[1.3beta-11] empty named export list throws syntax error

### ISSUE THREAD

**Issue by sungwoncho** [meteor/meteor#6281]
Using `export` sometimes throws syntax error. This happened in 1.3beta-11, but also in 1.3module-beta8. Did not check other releases.

Basically, the following throws an error:

```js
export {
};
```

```
W20160221-21:41:15.934(11)? (STDERR) /Users/mikecho/dev/meteor_projects/meteor-1.3-beta-module-bug/.meteor/local/build/programs/server/app/app.js:10
W20160221-21:41:15.934(11)? (STDERR) export {};                                                           // 1
W20160221-21:41:15.934(11)? (STDERR) ^^^^^^
W20160221-21:41:15.938(11)? (STDERR) SyntaxError: Unexpected reserved word
```

While the following does not:

```js
export {
  undefined
}
```

The following also does not throw:

```js
import something from 'something';

export {
  something
}
```

Reproduction: https://github.com/sungwoncho/meteor-1.3-beta-module-bug

**Comment by benjamn** [comment]
This seems to be a bug in Babel 6. Since empty `export` lists can be easily avoided, let's revisit this after the 1.3 release.

**Comment by abernix** [comment]
Closing this as it's fixed by #7033 (upcoming Meteor 1.3.3) which uses a new compiler for ES6 modules thus avoiding the Babel bug.  You can try it right now in the `1.3.3-beta.1` release by running this in your project:

    meteor update --release 1.3.3-beta.1

Thanks for reporting this issue!
